src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+
+    private double upperBase;
+    private double lowerBase;
+    private double height;
+
+    public double getUpperBase() {
+        return upperBase;
+    }
+
+    public double getLowerBase() {
+        return lowerBase;
+    }
+
+    public double getHeight() {
+        return height;
+    }
+
+    public void setUpperBase(double upperBase) {
+        this.upperBase = upperBase;
+    }
+
+    public void setLowerBase(double lowerBase) {
+        this.lowerBase = lowerBase;
+    }
+
+    public void setHeight(double height) {
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return ((upperBase + lowerBase) / 2) * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isosceles trapezoid, area: " + getArea()
+                + " sq.units, upperBase: " + upperBase + " units, lowerBase: "
+                + lowerBase + " units, height: " + height + " units, color: "
+                + getColor());
+
```

Comment:
remove the last empty line in all classes/interfaces/methods where it is present
```suggestion
```